File: SDKs/googleapis.com/shoppingcontent/v2/python/src/sdk/models/shared/accounttaxcustombatchresponseentry.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import accounttax as shared_accounttax
from ..shared import errors as shared_errors


@dataclass_json
@dataclasses.dataclass
class AccounttaxCustomBatchResponseEntry:
    r"""AccounttaxCustomBatchResponseEntry
    A batch entry encoding a single non-batch accounttax response.
    """
    
    account_tax: Optional[shared_accounttax.AccountTax] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('accountTax') }})
    batch_id: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('batchId') }})
    errors: Optional[shared_errors.Errors] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('errors') }})
    kind: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('kind') }})
    

File: SDKs/googleapis.com/content/v2/java/lib/src/main/java/openapisdk/models/shared/AccounttaxCustomBatchResponseEntry.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * AccounttaxCustomBatchResponseEntry
 * A batch entry encoding a single non-batch accounttax response.
**/
public class AccounttaxCustomBatchResponseEntry {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("accountTax")
    public AccountTax accountTax;
    public AccounttaxCustomBatchResponseEntry withAccountTax(AccountTax accountTax) {
        this.accountTax = accountTax;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("batchId")
    public Long batchId;
    public AccounttaxCustomBatchResponseEntry withBatchId(Long batchId) {
        this.batchId = batchId;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("errors")
    public Errors errors;
    public AccounttaxCustomBatchResponseEntry withErrors(Errors errors) {
        this.errors = errors;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("kind")
    public String kind;
    public AccounttaxCustomBatchResponseEntry withKind(String kind) {
        this.kind = kind;
        return this;
    }
}